### DISCUSSION TITLE
Making Folder listing settings more discoverable

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#878] [Feature Requests]
Multiple users have failed to find how to only the filenames. I think I should also add it to the settings.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by abatkin** [comment]
For me, my primary organizational structure - the way I find the file I want - is by filename, so the folder hierarchy is an important part of that (the primary exception is my "daily notes", which have their own hierarchy). I usually don't remember the exact filename (or path), so in my perfect world, there would be a quick and easy way to start typing and to quickly narrow down what I want (like vim's Ctrl-P plugin, fzf, etc). In many cases this means that I might start typing part of a folder name, and it would filter down to things within that folder (since that would match).

I know this isn't exactly in-line with "making folder listing settings more discoverable" but I wonder if it might somehow be a similar for users who arrange files in that way..
